pyresample/_formatting_html.py

```diff
@@ -244,25 +244,30 @@ def swath_area_attrs_section(area: 'geom.SwathDefinition') -> str: # noqa F821
         - Improve resolution estimation from lat/lon arrays. Maybe use CoordinateDefinition.geocentric_resolution?
 
     """
-    if isinstance(area.lons, np.ndarray) & isinstance(area.lats, np.ndarray):
-        # Calculate and estimated resolution from lats/lons in meter
-        area_name = "Arbitrary Swath"
-        resolution_y = np.mean(area.lats[0:-1, :] - area.lats[1::, :])
-        resolution_x = np.mean(area.lons[:, 1::] - area.lons[:, 0:-1])
-        resolution = np.mean(np.array([resolution_x, resolution_y]))
-        resolution = np.round(40075000 * resolution / 360, 1)
-        resolution_str = f"{resolution}/{resolution}"
+    if np.ndim(area.lons) == 1:
+        area_name = "1D Swath"
+        resolution_str = "{resolution}/{resolution}".format(resolution="Na")
+        height, width = "Na", "Na"
```

Comment:
Yeah you are right, I will just hard code it. I just took the code from the other cases but does not make sense here. Good idea with the "x" instead of "/" definitely makes more sense.

Well as mentioned somewhere before the resolution calculation in case of the attributes not having resolution set is just a rough estimate and therefore the circumference of the earth (40075000) is just hard coded.